test/parse.jl

```diff
@@ -21,7 +21,9 @@ docstr = """
 </bookstore>
 """
 
-xdoc = parse_string(docstr)
+for xdoc = (parse_string(docstr),
+            parse_file(joinpath(dirname(@__FILE__), "ex1.xml")),
+            parse_file(joinpath(dirname(@__FILE__), "ex1.xml"), C_NULL, 64)) # 64 == XML_PARSE_NOWARNING
```

Comment:
don't actually have a use case for that, would rather this just be a pretty thin wrapper around the C API